test/AutoDiff/validation-test/simple_math.swift

```diff
@@ -0,0 +1,408 @@
+// RUN: %target-run-simple-swift
+// NOTE(TF-813): verify that enabling forward-mode does not affect reverse-mode.
+// RUN: %target-run-simple-swift-forward-mode-differentiation
+// RUN: %target-swift-frontend -Xllvm -sil-print-after=differentiation %s -emit-sil -o /dev/null 2>&1 | %FileCheck %s
+// REQUIRES: executable_test
+
+import _Differentiation
+import StdlibUnittest
+import DifferentiationUnittest
+#if os(macOS)
```

Comment:
Can these ifdef'd `libc` imports be removed? If not, more conditions are necessary for Darwin:
```suggestion
#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
import Darwin.C
#else
import Glibc
#endif
```

If `libc` does need to be imported, perhaps `DifferentiationUnittest` could `@_exported import` it so individual tests don't need to. `DifferentiationUnittest` could `@_exported import _Differentiation`, too.

Reply:
It works without the libc. I have removed it.

Added `@_exported import _Differentiation`.